### ISSUE TITLE
Add Documentation for electron + react-dnd

### ISSUE THREAD

**Issue by eliw00d** [react-dnd/react-dnd#3501]
**Is your feature request related to a problem? Please describe.**
I am working on an Electron application that has react-dnd and also uses [Native File Drag & Drop](https://www.electronjs.org/docs/latest/tutorial/native-file-drag-drop). However, they don't interact well together. The Electron documentation has this snippet:

```javascript
document.getElementById('drag').ondragstart = (event) => {
  event.preventDefault()
  window.electron.startDrag('drag-and-drop.md')
}
```

But, preventing default prevents the HTML5 backend from working. If I remove that part but use startDrag, the application crashes. One or the other will work, but not both.

Basically, I want to be able to both drop items onto designated drop targets inside the application AND drop files outside of the application.

**Describe the solution you'd like**
Seeing as react-dnd and react-dnd-html5-backend [show up a lot in Electron applications](https://zeke.github.io/electron.atom.io/userland/dependencies/?q=react-dnd), it would be nice to have an example or some documentation around how to use both Native File Drag & Drop and useDrag and useDrop.

**Comment by nkjaiswal** [comment]
+1

**Comment by impactcolor** [comment]
@eliw00d did you ever find a solution to this? If so please share.

**Comment by eliw00d** [comment]
> @eliw00d did you ever find a solution to this? If so please share.

Unfortunately, no. I removed react-dnd altogether and ended up using HTML events directly.
